firefox-ios/firefox-ios-tests/Tests/ClientTests/MainMenu/MainMenuConfigurationUtilityTests.swift

```diff
@@ -0,0 +1,77 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import Common
+import Shared
+
+@testable import Client
+
+final class MainMenuConfigurationUtilityTests: XCTestCase {
+    var configUtility: MainMenuConfigurationUtility!
+    let windowUUID: WindowUUID = .XCTestDefaultUUID
+
+    override func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies()
+        configUtility = MainMenuConfigurationUtility()
+    }
+
+    override func tearDown() {
+        super.tearDown()
```

Comment:
let's move this to the last call in this method. See our guidelines here: https://github.com/mozilla-mobile/firefox-ios/wiki/Unit-Tests-Guidelines#1-setupteardown